package main.exception;

public class UserNotFoundException extends RuntimeException{
}
